internal/utils/utils.go

```diff
@@ -14,3 +17,17 @@ func ISliceToStrSlice(s []interface{}) (stringSlice []string, err error) {
 
 	return stringSlice, nil
 }
+
+func StructToJSONMap(s interface{}) map[string]interface{} {
```

Comment:
Have you seen the https://github.com/fatih/structs package or https://github.com/mitchellh/mapstructure

Reply:
Thank you, I have changed implementation